src/usb_cam.cpp

```diff
@@ -637,9 +637,6 @@ int UsbCam::read_frame()
 
       image_->stamp = stamp;
       break;
-    case IO_METHOD_UNKNOWN:
-      errno_exit("io method is unknown");
-      break;
```

Comment:
case IO_METHOD_UNKNOWN: is not duplicate

switch (buf.flags & V4L2_BUF_FLAG_TIMESTAMP_MASK) {
case V4L2_BUF_FLAG_TIMESTAMP_COPY:	 
case V4L2_BUF_FLAG_TIMESTAMP_UNKNOWN:
case IO_METHOD_USERPTR:
case IO_METHOD_UNKNOWN: <<---you want to delete it

Reply:
it is duplicate, look carefully:
```
525:  switch(io_) {
527:  case IO_METHOD_UNKNOWN: 
   ...
640:  case IO_METHOD_UNKNOWN: 
   ...
```

`switch (buf.flags & V4L2_BUF_FLAG_TIMESTAMP_MASK)` scope ends on `V4L_BUF_FLAG` cases.